app/Observers/ChannelObserver.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Observers;
+
+use App\Models\Channel;
+
+/**
+ * Observer on Channel Object
+ *
+ * These methods will be fired by Eloquent base events.
+ * - Before deleting we want to updte the data to the external services
+ */
+class ChannelObserver
+{
+    /**
+     * process before entitiy is removed
+     *
+     * trigger jobs of Channel
+     * that will sync and remove values from external services
+     *
+     * @param  Channel $channel
+     * @return void
+     */
+    public function deleting(Channel $channel)
+    {
+        app('ChannelService')->makeSyncJobsForExternalServices($channel);
+    }
```

Comment:
Add saved event.

Reply:
will be reviewed in the refactoring of external services https://github.com/StadGent/laravel_site_opening-hours/issues/93 https://github.com/StadGent/laravel_site_opening-hours/issues/92